src/main.go

```diff
@@ -35,6 +35,9 @@ func stderr(f string, a ...interface{}) {
 // hasUnrecognizedKeys finds unrecognized keys and warns about them on stderr.
 // returns false when no unrecognized keys were found, true otherwise.
 func hasUnrecognizedKeys(inCfg interface{}, refType reflect.Type) (warnings bool) {
+	if refType.Kind().String() == "ptr" {
```

Comment:
Err, maybe not. `refType` is a `reflect.Type`, but `reflect.Indirect()` takes a `reflect.Value`, and the `reflect.Type` interface doesn't have an `Indirect` function.